core/tests/protractor.conf.js

```diff
@@ -324,7 +324,9 @@ exports.config = {
         pathBuilder: function(currentSpec) {
           return currentSpec.fullName;
         },
-        captureOnlyFailedSpecs: true
+        captureOnlyFailedSpecs: true,
+        reportFailedUrl: true,
+        preserveDirectory: true
```

Comment:
This won't remove the existing dir. while starting the e2e tests?

Reply:
That is correct. We need to preserve screenshots across retries, so we can't delete the directory